### ISSUE TITLE
Documentation for 1.9.0

### ISSUE THREAD

**Issue by GFJHogue** [dockstore/dockstore#3583]
Similar to #3134 the following are a few points of confusion I ran into while going through the [Getting Started Guide](https://docs.dockstore.org/) (https://docs.dockstore.org/). Overall it was very clear, but here I'll list some issues I encountered.

### Getting Started with CWL
+ [Testing Locally](https://docs.dockstore.org/en/develop/getting-started/getting-started-with-cwl.html#testing-locally)
  - Should inform the reader to follow the step to *Install cwltool* when pointing them to [setup the Dockstore CLI locally](https://dockstore.org/quick-start) (https://dockstore.org/quick-start) since it is the *Optional* step.
  - For [setup the Dockstore CLI locally](https://dockstore.org/quick-start) `pip3 install -r requirements.txt` gives me errors depending on the initial conditions of the Python 3 environment. Could be due to my conda setup, but requiring strict versions in `requirements.txt` may be overkill and cause issues with installing the dependencies.

### Getting Started with WDL
+ [Testing Locally](https://docs.dockstore.org/en/develop/getting-started/getting-started-with-wdl.html#testing-locally)
  - The Java/Cromwell command at the end of this section is a bit awkward. Could use a label or a comment to clarify, like "Cromwell command run by dockstore".

### Tools
+ [Quick Registration of Alternate Tools/Manual Registration of Tools](https://docs.dockstore.org/en/develop/getting-started/dockstore-tools.html#quick-registration-of-alternate-tools) & [Manual](https://docs.dockstore.org/en/develop/getting-started/dockstore-tools.html#manual)
  - This page in 3 places refers to a "plus button" on the bottom-right of the UI, but I don't see any. The Create button with the plus sign appears to now be on the top-left of the page and I believe that's what it meant.

┆Issue is synchronized with this [Jira Story](https://ucsc-cgl.atlassian.net/browse/DOCK-1432)
┆Issue Type: Story
┆Fix Versions: Dockstore 1.9.X
┆Sprint: Sprint 42 - Quahog
┆Issue Number: DOCK-1432
┆Epic: Dockstore documentation

**Comment by denis-yuen** [comment]
`Could be due to my conda setup, but requiring strict versions in requirements.txt may be overkill and cause issues with installing the dependencies.` 
Not so familiar with Python, the idea here was cwltool can change a lot from version to version (they don't use semantic versioning) so we wanted to recommend a specific version we have tested and run through tool tester

**Comment by GFJHogue** [comment]
@denis-yuen I didn't have any issue with the specified version of `cwltool`, but rather some of its dependencies (`ruamel`, `psutil`) with versions set in `requirements.txt` threw errors during the `pip3 install`.

**Comment by garyluu** [comment]
A few things:

Installing via those instructions also failed on my machine (it needed python3-dev)

Story: 
1. I'm guessing at the beginning we only told people to install a specific cwltool version
2. Some things happened and we needed to lock the ruamel (or something else) to a specific version (very short requirements.txt file)
3. cwltool fixed their dependencies on their end and everything should just work if we go back to 1
4. I brought this up and we voted against going back to 1 (😢 )
5. We went the opposite direction and now it's this requirements.txt
